### PR TITLE
Test case to add License.

### PR DESCRIPTION
**Pull request by BDK-1990**
Please review test case to add License

### CODE REVIEW

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/LicenseTests.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class LicenseTests extends TestBase {
+    @Test
+    public void addNewLicense() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("Admin", "admin123", true, null);
+        HeaderPage headerPage = new HeaderPage(driver);
+        headerPage.selectMenu(MenuOptions.LICENSES);
+        LicensePage licensePage = new LicensePage(driver);
+        licensePage.saveNewLicense("ISO-2005");
+       // String uuid = UUID.randomUUID().toString();
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/tests/LicenseTests.java]: please uncomment these lines and remove line 17

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/EducationTests.java [hunk 2]
```diff
@@ -14,12 +14,12 @@ public class EducationTests extends TestBase {
 @Test
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/tests/EducationTests.java:14]: revert all changes you made on this class

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/LicensePage.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+
+import java.util.List;
+
+public class LicensePage extends PageBase {
+    WebDriver driver;
+   private final String idLicAddBtn = ".oxd-button--secondary";
+
+    // private final String idLicTxtLevel="div[class='oxd-form-row'] input[class*='oxd-input']";
```
Comment (RahulNair92) [inline src/test/java/ca/qaguru/oranghrmbatch27/pages/LicensePage.java]: remove comments blank lines

**Review by RahulNair92 — CHANGES_REQUESTED** [review]
please fix teh review comments

File: src/test/java/ca/qaguru/oranghrmbatch27/tests/LicenseTests.java (hunk 1 above)
Comment (BDK-1990) [reply src/test/java/ca/qaguru/oranghrmbatch27/tests/LicenseTests.java]: Done

File: src/test/java/ca/qaguru/oranghrmbatch27/pages/LicensePage.java (hunk 3 above)
Comment (BDK-1990) [reply src/test/java/ca/qaguru/oranghrmbatch27/pages/LicensePage.java]: Removed.

**Review by RahulNair92 — APPROVED** [review]

**Pull request merged**